src/datanode/src/store/gcs.rs

```diff
@@ -36,6 +37,8 @@ pub(crate) async fn new_gcs_object_store(gcs_config: &GcsConfig) -> Result<Objec
         .credential_path(gcs_config.credential_path.expose_secret())
         .endpoint(&gcs_config.endpoint);
 
+    builder.http_client(build_http_client()?);
```

Comment:
Why not use chain calls, like the example below:
```
builder
        .root(&root)
        .bucket(&gcs_config.bucket)
        .scope(&gcs_config.scope)
        .credential_path(gcs_config.credential_path.expose_secret())
        .endpoint(&gcs_config.endpoint)
        .http_client(build_http_client()?);
```

Reply:
Forgot to modify it. The `build_http_client` function is extracted later.